pkg/addons/data/Deployment.apps/openshift-azure-monitoring/metrics-bridge.yaml

```diff
@@ -7,79 +7,72 @@ metadata:
   namespace: openshift-azure-monitoring
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: metrics-bridge
   strategy:
-    type: RollingUpdate
     rollingUpdate:
```

Comment:
If you are relying on defaults then you can also remove these rolling parameters.